api/build.gradle

```diff
@@ -29,6 +32,19 @@ idea {
     }
 }
 
+sonarqube {
+    properties {
+        property "sonar.pitest.mode", "reuseReport"
```

Comment:
Root build.gradle has already sonarqube config. May be that's the best place for these props as well?